<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>15</y>
      <width>841</width>
      <height>571</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;的&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_food">
     <attribute name="title">
      <string>学生信息表</string>
     </attribute>
     <widget class="QTableView" name="tableView_food">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>841</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
     <zorder>tableView_food</zorder>
     <zorder>but_choose</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>600</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>请输入要查找的学生姓名或者ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_choose">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>查看全表</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>610</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="but_search">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_delete">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>370</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除选中行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_id_down">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>540</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>id降序排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_submit">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>220</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>提交修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_add">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>140</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>添加记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_id_up">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>450</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>id升序排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_back">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>300</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>撤销修改</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>1</y>
      <width>1271</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_comment">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>备注：</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>tabWidget</zorder>
   <zorder>label</zorder>
   <zorder>but_choose</zorder>
   <zorder>lineEdit_search</zorder>
   <zorder>but_search</zorder>
   <zorder>but_delete</zorder>
   <zorder>but_id_down</zorder>
   <zorder>but_submit</zorder>
   <zorder>but_add</zorder>
   <zorder>but_id_up</zorder>
   <zorder>but_back</zorder>
   <zorder>tableView_food</zorder>
   <zorder>label_comment</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
